### PR TITLE
Bug 1907280: Adds quickstart for ocs install tour

### PR DESCRIPTION
**Pull request by vbnrh**
Depends on https://github.com/openshift/console-operator/pull/490

Signed-off-by: Vineet Badrinath <vbadrina@redhat.com>

### CODE REVIEW

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (spadgett) [inline quickstarts/ocs-install-tour-quickstart.yaml:5]: This is missing the required cluster profile annotations.

```
  annotations:
    include.release.openshift.io/ibm-cloud-managed: "true"
    include.release.openshift.io/self-managed-high-availability: "true"
    include.release.openshift.io/single-node-developer: "true"```

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (spadgett) [inline quickstarts/ocs-install-tour-quickstart.yaml:8]: It should be OpenShift throughout.

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (spadgett) [inline quickstarts/ocs-install-tour-quickstart.yaml:33]: Is this a stray backtick at the end of the sentence?

**Review by spadgett — COMMENTED** [review]
Hi, @vbnrh. It looks like this might not have been reviewed for content. We should get that done if not.

/cc @beanh66

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (vbnrh) [reply quickstarts/ocs-install-tour-quickstart.yaml:5]: added. thanks

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (vbnrh) [reply quickstarts/ocs-install-tour-quickstart.yaml]: fixed

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (vbnrh) [reply quickstarts/ocs-install-tour-quickstart.yaml]: removed

**Comment by spadgett** [comment]
@bmignano @beanh66 Would you be able to help us review the quick start for content?

**Review by abigaeljamie — COMMENTED** [review]
I was able to review a few steps and provide revisions, but my UI stopped matching the instructions once I reached the step about Operator Subscription form. I sent a meeting invite to Yuval for Monday to walk through this.

**Review by jhadvig — COMMENTED** [review]
After the comments from @abigaeljamie 's comments are addressed I would also keep I on https://github.com/openshift/console-operator/pull/493 cause its adding an additional annotation to the QuickStarts.
Please keep an eye on the PR and if its merged before your's PR, please add the annotation.

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (vbnrh) [reply quickstarts/ocs-install-tour-quickstart.yaml]: addressed

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (vbnrh) [reply quickstarts/ocs-install-tour-quickstart.yaml]: addressed

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (vbnrh) [reply quickstarts/ocs-install-tour-quickstart.yaml:15]: addressed

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (vbnrh) [reply quickstarts/ocs-install-tour-quickstart.yaml]: > After the comments from @abigaeljamie 's comments are addressed I would also keep I on #493 cause its adding an additional annotation to the QuickStarts.
> Please keep an eye on the PR and if its merged before your's PR, please add the annotation.

thanks for the heads up. IDo you want me to add the annotation now itself before your PR is merged ?

**Comment by vbnrh** [comment]
> After the comments from @abigaeljamie 's comments are addressed I would also keep I on #493 cause its adding an additional annotation to the QuickStarts.
> Please keep an eye on the PR and if its merged before your's PR, please add the annotation.

@jhadvig thanks for the heads up. IDo you want me to add the annotation now itself before your PR is merged ?

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (vbnrh) [reply quickstarts/ocs-install-tour-quickstart.yaml]: addressed

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (vbnrh) [reply quickstarts/ocs-install-tour-quickstart.yaml]: addressed

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (vbnrh) [reply quickstarts/ocs-install-tour-quickstart.yaml]: addressed

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (vbnrh) [reply quickstarts/ocs-install-tour-quickstart.yaml]: the form shows up right before before we click on installation. i hope @yuvalgalanti has cleared it up

**Comment by jhadvig** [comment]
@vbnrh yes, please add it.

**Comment by vbnrh** [comment]
> @vbnrh yes, please add it.

done

**Comment by jhadvig** [comment]
Thanks @vbnrh 
Will wait for final review from @abigaeljamie before tagging.

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (abigaeljamie) [reply quickstarts/ocs-install-tour-quickstart.yaml:49]: 6. After creating the storage cluster, you'll be brought to the storage cluster details page. The state of the cluster appears beside the cluster name. Wait for the state of the cluster to change from **Progressing** to **Ready.** This may take a few minutes.

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (abigaeljamie) [inline quickstarts/ocs-install-tour-quickstart.yaml:50]:
```suggestion
      7. Once the state of the cluster is **Ready**, you can start using the OpenShift container storage resources (storage classes and the bucket classes) to allow OCS to manage your storage.
```

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (abigaeljamie) [inline quickstarts/ocs-install-tour-quickstart.yaml:53]:
```suggestion
        #### Verify that the Storage Cluster is created:
```

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (abigaeljamie) [inline quickstarts/ocs-install-tour-quickstart.yaml:55]: On the storage cluster details page, is the state of the cluster **Ready** ?

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (abigaeljamie) [inline quickstarts/ocs-install-tour-quickstart.yaml:58]:
```suggestion
      success: Great work! You installed the OpenShift Container Storage operator.
```

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (abigaeljamie) [inline quickstarts/ocs-install-tour-quickstart.yaml:60]:
```suggestion
  conclusion: Congratulations! The OpenShift Container Storage operator is ready to use. To learn how you can manage your storage space effectively, take the Getting Started With OpenShift Container Storage quick start.
```

**Review by abigaeljamie — CHANGES_REQUESTED** [review]
All set, @yuvalgalanti ! Let me know if you have any questions 🙂 Thanks!

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (vbnrh) [reply quickstarts/ocs-install-tour-quickstart.yaml:60]: fixed

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (vbnrh) [reply quickstarts/ocs-install-tour-quickstart.yaml:58]: done

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (vbnrh) [reply quickstarts/ocs-install-tour-quickstart.yaml]: done

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (vbnrh) [reply quickstarts/ocs-install-tour-quickstart.yaml]: done

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (vbnrh) [reply quickstarts/ocs-install-tour-quickstart.yaml]: changed

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (vbnrh) [reply quickstarts/ocs-install-tour-quickstart.yaml]: done

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (vbnrh) [reply quickstarts/ocs-install-tour-quickstart.yaml]: done

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (vbnrh) [reply quickstarts/ocs-install-tour-quickstart.yaml]: done

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (vbnrh) [reply quickstarts/ocs-install-tour-quickstart.yaml:25]: added

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (vbnrh) [reply quickstarts/ocs-install-tour-quickstart.yaml]: fixed

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (vbnrh) [reply quickstarts/ocs-install-tour-quickstart.yaml]: fixed

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (vbnrh) [reply quickstarts/ocs-install-tour-quickstart.yaml]: i've changed it to your suggestion

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (vbnrh) [reply quickstarts/ocs-install-tour-quickstart.yaml]: done

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (vbnrh) [reply quickstarts/ocs-install-tour-quickstart.yaml]: fixed

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (vbnrh) [reply quickstarts/ocs-install-tour-quickstart.yaml]: changed to "Install the OpenShift Container Storage operator"

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (vbnrh) [reply quickstarts/ocs-install-tour-quickstart.yaml]: good idea ! i have removed the step

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (vbnrh) [reply quickstarts/ocs-install-tour-quickstart.yaml]: fixed

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (vbnrh) [reply quickstarts/ocs-install-tour-quickstart.yaml]: done

**Comment by jhadvig** [comment]
@abigaeljamie please approve the PR when you are satisfied with all the changes. Will tag for merge afterwards :)

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (abigaeljamie) [inline quickstarts/ocs-install-tour-quickstart.yaml:44]:
```suggestion
      A storage cluster includes a Ceph cluster, Multi cloud gateway, and all the required storage and commute resources.
``` 

There was just a missing comma and extra space here, so I added that above in the edit.

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (abigaeljamie) [inline quickstarts/ocs-install-tour-quickstart.yaml:25]: It looks like we're missing the new step #1 here (entering the admin perspective). It's included in the second task, but it looks like it got left out here. Here is the line: 

1. 1. Enter the administrator perspective: In the main navigation, click the dropdown menu and select **Administrator.**

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (abigaeljamie) [inline quickstarts/ocs-install-tour-quickstart.yaml:53]:
```suggestion
          - **Internal**: Can be used on any platform except bare metal
``` 
Removed the period

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (abigaeljamie) [inline quickstarts/ocs-install-tour-quickstart.yaml:55]:
```suggestion
          - **External**: Used for external RHCS (Red Hat Cluster Suite) Cluster
```

**Review by abigaeljamie — CHANGES_REQUESTED** [review]
There were just a few small leftover edits that didn't make it in, so I added those again. Otherwise, it looks great!

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (vbnrh) [reply quickstarts/ocs-install-tour-quickstart.yaml:55]: done

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (vbnrh) [reply quickstarts/ocs-install-tour-quickstart.yaml:53]: done

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (vbnrh) [reply quickstarts/ocs-install-tour-quickstart.yaml]: done

File: quickstarts/ocs-install-tour-quickstart.yaml
Comment (vbnrh) [reply quickstarts/ocs-install-tour-quickstart.yaml]: done

**Review by abigaeljamie — APPROVED** [review]
LGTM! 🎉 Thank you :)

**Review by jhadvig — APPROVED** [review]
/lgtm
/approve

**Comment by jhadvig** [comment]
@vbnrh we need a BZ for this since its past feature freeze

**Pull request merged**
